app/policies/person_policy.rb

```diff
@@ -0,0 +1,5 @@
+class PersonPolicy < ApplicationPolicy
+  def admin?
+    person.present? && person.admin?
```

Comment:
What's `person` here? Is this a global method that we have access to? Should it be `current_person`?